src/pages/LoanFinding/LoanFinding.vue

```diff
@@ -262,22 +284,31 @@ export default {
 	},
 	methods: {
 		async getRecommendedLoans() {
-			const { loans } = await runLoansQuery(
-				this.apollo,
-				{ pageLimit: 12 },
-				FLSS_ORIGIN_LEND_BY_CATEGORY
-			);
+			let loans = [];
+			if (this.enableLoanRecommendations) {
+				const response = await runRecommendationsQuery(this.apollo, {
+					userId: this.userInfo?.id,
```

Comment:
Do we also need to pass in `null` here if there is no ID (like in `created`)?

Reply:
done